resources/views/about/index.blade.php

```diff
@@ -20,11 +20,9 @@
         <div class="card card-body">
         <img class="about-logo" src="/img/md-blk-logo.png">
         <hr>
-        <div>{{__('ProcessMaker 4')}} Beta 10 Release Candidate 1</div>
+        <div>{{__('ProcessMaker 4')}} Beta 10 Release Candidate 2</div>
         <hr>
-        <a href="https://docs.google.com/forms/d/e/1FAIpQLScnYje8uTACYwp3VxdRoA26OFkbfFs6kuXofqY-QXXsG-h9xA/viewform" target="_blank">{{__('Report an issue')}}  <i class="fas fa-caret-right fa-lg float-right mr-1"></i></a>
-        <hr>
-        <a href="https://github.com/ProcessMaker/bpm" target="_blank">{{__('Get Help')}}  <i class="fas fa-caret-right fa-lg float-right mr-1"></i></a>
+        <a href="https://github.com/ProcessMaker/processmaker/issues/new" target="_blank">{{__('Report an issue')}}  <i class="fas fa-caret-right fa-lg float-right mr-1"></i></a>
```

Comment:
Recommend changing this to "Report Issue" to match the formal tone of the other menu items.